laserstudio/widgets/viewer.py

```diff
@@ -52,6 +52,8 @@ class Mode(int, Enum):
     mode_changed = pyqtSignal(int)
     # Signal emitted when the mouse has moved in scene
     mouse_moved = pyqtSignal(float, float)
+    # Signal emitted when the follow stage sight option changed
+    followStageSightChanged = pyqtSignal(bool)
```

Comment:
I would name it `follow_stage_sight_changed` to respect PEP8 (and to be coherent with precedent names)